hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/AbstractAbfsIntegrationTest.java

```diff
@@ -383,4 +391,34 @@ protected AbfsDelegationTokenManager getDelegationTokenManager()
       throws IOException {
     return getFileSystem().getDelegationTokenManager();
   }
+
+  /**
+   * Generic assert method.
+   *
+   * @param operation     operation being asserted.
+   * @param expectedValue value that is expected.
+   * @param actualValue   value that is actual.
+   */
+  protected void assertValues(String operation, long expectedValue,
```

Comment:
too vague a name and no obvious difference with assertEquals. 

Propose:
*  leave existing test alone, 
and then one of
* use Junit assertEquals()
* or Assertions.assertThat(object).describedAs().equals().